applications/RomApplication/tests/test_calculate_rom_basis_output_process.py

```diff
@@ -33,7 +33,6 @@ def testCalculateRomBasisOutputProcess(self):
             "snapshots_interval": 1.0,
             "nodal_unknowns": ["TEMPERATURE"],
             "rom_basis_output_format": "json",
-            "rom_basis_output_name": "RomParameters",
```

Comment:
Same comment. I think we should keep this user-definable.

Reply:
The issue here is, even when the field "rom_basis_output_name" seemed to provide the use the freedom to choose the name of the file to store the ROM information; in reality, the name RomPameters is hardcoded elsewhere within the workflow. Therefore, choosing a different name here, will invariably lead to a FILE_NOT_FOUND error.

In a future PR, we shall allow to name the file differently, which might come handy for "non-standard" simulations like coupled problems. Of course the solution shall involve other sections of the code and probably should happen within the RomManager #10763